shell/platform/android/io/flutter/plugin/platform/PlatformPlugin.java

```diff
@@ -531,6 +539,9 @@ private CharSequence getClipboardData(PlatformChannel.ClipboardContentFormat for
       return null;
     } catch (FileNotFoundException e) {
       return null;
+    } catch (IOException e) {
+      Log.w(TAG, "Failed to close AssetFileDescriptor while accessing clipboard data.", e);
+      return null;
```

Comment:
I think this should return charSequence not null. If we have the file contents and fail to close I think they should still be returned.

Reply:
Changed!